vispy/visuals/visual.py

```diff
@@ -30,6 +30,11 @@ class Visual(object):
     bounds_change : Event
         Emitted when the bounds of the visual have changed.
 
+    Notes
+    -----
+    -   When used in the scenegraph, all Visual classes are mixed with
+    vispy.scene.Node in order to implement the methods, attributes and
```

Comment:
Please open an issue, yeah, and please change this in your PR so we at least start making things better incrementally :)